docs/reference/requirement-sets/powerpoint-api-requirement-sets.md

```diff
@@ -0,0 +1,58 @@
+---
+title: PowerPoint JavaScript API requirement sets
+description: ''
+ms.date: 07/17/2019
+ms.prod: powerpoint
+localization_priority: Normal
+---
+
+# PowerPoint JavaScript API requirement sets
+
+Requirement sets are named groups of API members. Office Add-ins use requirement sets specified in the manifest or use a runtime check to determine whether an Office host supports APIs that an add-in needs. For more information, see [Office versions and requirement sets](/office/dev/add-ins/develop/office-versions-and-requirement-sets).
+
+The following table lists the PowerPoint-specific requirement sets, the Office host applications that support those requirement sets, and the build versions or availability date.
+
+|  Requirement set  |  Office on Windows<br>(connected to Office 365 subscription)  |  Office on iPad<br>(connected to Office 365 subscription)  |  Office on Mac<br>(connected to Office 365 subscription)  | Office on the web |
+|:-----|-----|:-----|:-----|:-----|:-----|
+| PowerPointApi 1.1 | Version 1903 (Build 11425.20204) or later | 2.24 or later | 16.24 or later | May 2019 |
+
+## Office Common API requirement sets
+
+Most of the PowerPoint Add-in functionality comes from the Common API set. For information about Common API requirement sets, see [Office Common API requirement sets](office-add-in-requirement-sets.md).
+
+## PowerPoint JavaScript API 1.1
+
+PowerPoint JavaScript API 1.1 contains a single API to create a new presentation. For details about the API, see [JavaScript API for PowerPoint](../../powerpoint/powerpoint-add-ins.md).
+
+## Runtime requirement support check
+
+During the runtime, add-ins can check if a particular host supports an API requirement set by doing the following.
+
+```js
+if (Office.context.requirements.isSetSupported('PowerPointApi', "1.1") === true) {
+  // Perform actions.
+}
+else {
+  // Provide alternate flow/logic.
+}
+```
+
+## Manifest-based requirement support check
+
+Use the Requirements element in the add-in manifest to specify critical requirement sets or API members that your add-in must use. If the Office host or platform doesn't support the requirement sets or API members specified in the Requirements element, the add-in won't run in that host or platform, and won't display in My Add-ins.
+
+The following code example shows an add-in that loads in all Office host applications that support the OneNoteApi requirement set, version 1.1.
+
+```xml
+<Requirements>
+   <Sets DefaultMinVersion="1.1">
+      <Set Name="PowerPointApi" MinVersion="1.1"/>
+   </Sets>
+</Requirements>
+```
+
+## See also
+
```

Comment:
Consider adding a link for `[PowerPoint JavaScript API reference documentation](/javascript/api/powerpoint)`

Reply:
For consistency, would be good to add a link for `[OneNote JavaScript API reference documentation](/javascript/api/onenote)` to the [OneNote requirement sets article](https://docs.microsoft.com/en-us/office/dev/add-ins/reference/requirement-sets/onenote-api-requirement-sets) as well.